weld-python/tests/weld/lazy/test_weldfunc.py

```diff
@@ -0,0 +1,51 @@
+
+from weld import *
+from weld.lazy import WeldLazy, weldfunc
+from weld.types import *
+
+def test_weldfunc_simple():
```

Comment:
Can you write a test with 2 or more dependencies as well? And a longer chain with a single dependency each?